modules/pom.xml

```diff
@@ -1,120 +1,134 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>com.devonfw.java.dev</groupId>
-    <artifactId>devon4j</artifactId>
-    <version>dev-SNAPSHOT</version>
-  </parent>
-  <artifactId>devon4j-modules</artifactId>
-  <packaging>pom</packaging>
-  <name>${project.artifactId}</name>
-  <description>Reusable modules of devon4j.</description>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>com.devonfw.java.dev</groupId>
+		<artifactId>devon4j</artifactId>
+		<version>dev-SNAPSHOT</version>
+	</parent>
+	<artifactId>devon4j-modules</artifactId>
+	<packaging>pom</packaging>
+	<name>${project.artifactId}</name>
+	<description>Reusable modules of devon4j.</description>
```

Comment:
could you follow devonfw code style? Your IDE should be preconfigured with indendation by 2 spaces. Just hit [Ctrl][Shift][f]. This will help use to prevent diff wars in git. I am struggling often when doing reviews because of that when there are tons of lines in the diff that actually did not change but people where just using different formatters. I thought this was solved with `devonfw-ide`. Seems we still have to invest more on this...

Reply:
i did that CTRL+SHIFT+F but still it is same . let me try one more time